environments/openshift/jaeger.jsonnet

```diff
@@ -0,0 +1,92 @@
+local k = import 'ksonnet/ksonnet.beta.4/k.libsonnet';
+
+local app =
+  (import '../kubernetes/jaeger.libsonnet') + {
+    jaeger+:: {
+      namespace:: '${NAMESPACE}',
+      image:: '${IMAGE}:${IMAGE_TAG}',
+      replicas:: '${{REPLICAS}}',
+
+      queryService+: {
+        metadata+: {
+          annotations+: {
+            'service.alpha.openshift.io/serving-cert-secret-name': 'jaeger-tls',
```

Comment:
`jaeger-query-tls` would be better, as we might in the future have one for the collector as well, for the communication between agent and collector.